package.json

```diff
@@ -100,6 +101,7 @@
     "json-refs": "^2.1.5",
     "jwt-decode": "^2.2.0",
     "lodash": "^4.13.1",
+    "md5": "^2.2.1",
```

Comment:
Is this dependency really needed? can't we use the Node.js `crypto` module to generate the hash (https://stackoverflow.com/a/19237140).